arrays/arrays.c

```diff
@@ -118,11 +204,20 @@ void arr_remove(Array *arr, char *element) {
 
   // Search for the first occurence of the element and remove it.
   // Don't forget to free its memory!
-
+  int i = 0;
+  while(!is_same(arr->elements[i], element))
+  {
+    i++;
+  }
+  free(arr->elements[i]);
```

Comment:
What happens if the element you're looking to remove isn't present in the array?